Dynatrace/dynatrace-operator
# Add feature flag for ignoring proxy within oneAgent and activeGate

# Description
Currently we have the problem, when a proxy is set in the custom resource, it will get applied to all the components (operator, oneAgent and activeGate). This is so far intended, but if there is an activeGate with the purpose of routing agents traffic, then the oneAgent will also try to connect to the activeGate using the proxy, which causes problems

## How can this be tested?
Setup the operator and set a proxy in the customResource. This proxy will get applied to dtclient, oneAgents and activeGate (if defined in the same CR). If you add one or both featureFlags

- "operator.dynatrace.com/feature-activegate-ignore-proxy=true"
- "operator.dynatrace.com/feature-oneagent-ignore-proxy=true"

proxy should not be set for activeGate and/or oneAgent

## Checklist
